WTM.js

```diff
@@ -66,7 +66,7 @@ const executeRequest = async (object) => {
         ret.status = api_response.status;
         ret.text = await api_response.text();
         try {
-            ret.json = JSON.parse(ret.text);
+            ret.json = ret.text ? JSON.parse(ret.text) : ret.text;
```

Comment:
@Pankti2608 we don't want to set ret.json = ret.text.. it text cannot be parsed, ret.json should remain NULL.